spec/spec.ddoc

```diff
@@ -5,6 +5,23 @@ DDOC_BLANKLINE = $(DIVC blankline)
 
 SUBNAV=$(SUBNAV_SPEC)
 SUBNAV_SPEC=
+<style type="text/css">
+body {
+  counter-reset: para h3;
+}
+p:before {
+  counter-increment: para;
+  content: counter(h3) "." counter(para) ": ";
+}
+h3:before {
+  counter-increment: h3;
+  content: "" counter(h3) ": ";
+  counter-reset: para;
+}
+h3 {
+  counter-reset: para;
+}
+</style>
```

Comment:
This is not the correct place for this. CSS rules are global, adding them inline doesn't make sense.

1. Wrap `SUBNAV_SPEC` in a `div` or something else that gives it a unique id or class (e.g. `id="subnav-spec"`).
2. In style.css, add these rules, but prefixed with `#subnav-spec` (i.e. `#subnav-spec p:before { ... }`).